tasks/Nazyrov_A_Min_value_of_vector_elements/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# Nazyrov_A_Min_value_of_vector_elements task
```

Comment:
CMakeLists is not needed here. The task is automatically hooked by the infrastructure